lib/vty.c

```diff
@@ -2141,6 +2152,23 @@ static void vtysh_accept(struct event *thread)
 		close(sock);
 		return;
 	}
+	/* Setting the SND buffer size to 16MB to increase the buffer size
+	 * the socket can hold before sending it to VTY shell
+	 */
+	uint32_t sndbufsize = BUFFER_SIZE_DEFAULT * BUFFER_SIZE_DEFAULT;
```

Comment:
How it's better/worse if we use even bigger buffer sizes? I'm asking if it would be worth setting this dynamically at all?

Reply:
yes, agree. it'd be better to make the vty decisions independent of the underlying buffer module

Reply:
there was no additional difference in the buffer memory used beyond this send/receive buffer size since the rmem and wmem limit is 16MB.

Reply:
I don't think that answered the question.
Please don't bind the vty lib module's sizing to values from the generic buffer module. Define values for use by vty directly.

Reply:
I have changed that. Not using any sizing values from buffer module directly now. Brought it back to the existing code.